src/Cake.Npm/AddUser/NpmAddUserSettingsExtensions.cs

```diff
@@ -0,0 +1,91 @@
+﻿namespace Cake.Npm.AddUser
+{
+    using System;
+
+    /// <summary>
+    /// Extensions for <see cref="NpmAddUserSettings"/>.
+    /// </summary>
+    public static class NpmAddUserSettingsExtensions
+    {
+        /// <summary>
+        /// Instructs npm to add user for the given registry.
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <param name="registry">The registry for the user.</param>
+        /// <returns>The <paramref name="settings"/> instance with <see cref="NpmAddUserSettings.Registry"/> set to <paramref name="registry" />.</returns>
+        public static NpmAddUserSettings FromRegistry(this NpmAddUserSettings settings, Uri registry)
```

Comment:
As this adds an user for a specific registry this method should be called `ForRegistry`.